tests/ecosystem/deployment/test_deployment.py

```diff
@@ -78,3 +82,29 @@ def test_deployment(pvc_factory, pod_factory):
 
     if teardown:
         log.info("Cluster will be destroyed during teardown part of this test.")
+
+
+@azure_platform_required
+def test_azure_storageaccount():
+    """
+    Testing that Azure storage account, post deployment.
+
+    Testing for property 'allow_blob_public_access' to be 'false'
+    """
+    azure = azure_utils.AZURE()
+    storage_account_names = azure.get_storage_accounts_names()
+    for storage in storage_account_names:
+        if "noobaaaccount" in storage:
+            property = str(azure.get_storage_account_properties(storage))
```

Comment:
one more thing you need to consider while looking for `noobaaaccount` string in the list of storage accounts, which is to look for the noobaa storage account for the current cluster where this validation is being done. For example, when you have 2 azure cluster (azure-1 and azure-2) and this code is running on azure-2 then the validation should be done only for the azure-2 cluster noobaa storage account! This cluster name details can be found in the storage account properties itself IIUC

Reply:
added filter with resource group to get the storage account of particular deployment only.